blog/2023-01-25-zw448.md

```diff
@@ -0,0 +1,12 @@
+---
+title: About zw448
+authors:
+  name: Jamie Wang
+  title: Connective Media
+tags: MS in Connective Media
```

Comment:
👋 Congrats! This was the first change to break the site - this should have stayed as `tags: [intro]`. You wouldn't have known that anyway.

Nothing for you to do though, I'll quickly fix it. Just wanted to mention it